cmd/termui/provider/logProvider.go

```diff
@@ -65,29 +69,26 @@ func openWebSocket(scheme string, address string, profile logger.Profile) (*webs
 	return conn, nil
 }
 
-// StartListeningOnWebSocket will listen if a new log message is received and will display it
-func StartListeningOnWebSocket(presenter PresenterHandler) {
-	go func() {
-		for {
-			msgType, message, err := webSocket.ReadMessage()
-			if msgType == websocket.CloseMessage {
-				return
-			}
-			if err == nil {
-				writeMessage(presenter, message)
-				continue
-			}
-
-			_, isConnectionClosed := err.(*websocket.CloseError)
-			if !isConnectionClosed {
-				log.Error("termui websocket error", "error", err.Error())
-			} else {
-				log.Debug("termui websocket terminated", "error", err.Error())
-			}
+// startListeningOnWebSocket will listen if a new log message is received and will display it
+func startListeningOnWebSocket(presenter PresenterHandler) {
+	for {
+		msgType, message, err := webSocket.ReadMessage()
+		if msgType == websocket.CloseMessage {
+			return
+		}
+		if err == nil {
+			writeMessage(presenter, message)
+			continue
+		}
 
-			time.Sleep(retryDuration)
+		_, isConnectionClosed := err.(*websocket.CloseError)
+		if !isConnectionClosed {
+			log.Error("termui websocket error", "error", err.Error())
```

Comment:
maybe you can write this error in termui log because

Reply:
done